### ISSUE TITLE
Plateau Neuron Model for Simulating Dendritic Plateau Potentials 

### ISSUE THREAD

**Issue by kds300** [lava-nc/lava#775]
### User story
I want a neuron model which emulates a simplified version of dendritic plateau potentials. The model will take two separate inputs, one to the "soma" and another to the  "dendrite". Both the soma and dendrite  potentials will follow sub-threshold LIF dynamics. When the dendrite potential crosses its threshold it resets and sends the neuron to the UP state. The  UP state will last for a specified duration of time steps. During this time, the soma is able to spike and reset if its potential crosses its threshold. If the neuron is in the DOWN state, the soma potential will follow subthreshold LIF dynamics regardless of whether it crosses the threshold.

In summary, the plateau neuron couples two separate inputs such that a spike is generated by sufficiently strong input from one followed by the other within a short enough time period.
 
### Conditions of satisfaction
-  Users can specify two separate thresholds for the dendrite and soma, respectively.
-  Users can specify two separate voltage decays.
-  Users can specify the duration of the UP state, in time steps. 

### Acceptance tests
-   The neuron only spikes if it is in the UP state

**Comment by PhilippPlank** [comment]
Thank you for this effort!
